cottoncandy/interfaces.py

```diff
@@ -204,6 +204,11 @@ def create_bucket(self, bucket_name, acl=DEFAULT_ACL):
         self.connection.create_bucket(Bucket=bucket_name, ACL=acl)
         self.set_bucket(bucket_name)
 
+    def rm_bucket(self, bucket_name, acl=DEFAULT_ACL):
```

Comment:
Good idea! I think you can remove the ``acl=DEFAULT_ACL`` kwarg. Also, this will crash for non-empty buckets, maybe put a try / except and make the except raise a more informative error (i.e. the bucket is not empty).